graphql/benchmark_test.go

```diff
@@ -27,20 +27,38 @@ func pagintedQueryWithFilterBenchmark(b *testing.B, n int, batchFunc bool) {
 		items[i] = Item{Id: int64(i), FilterText: filterTexts[i%5], String: "a"}
 	}
 	if !batchFunc {
-		inner.FieldFunc("innerConnectionWithFilter", func() []Item {
-			return items
-		}, schemabuilder.Paginated,
-			schemabuilder.FilterField("foo",
-				func(ctx context.Context, i Item) string {
-					return i.FilterText
-				},
-			),
-			schemabuilder.FilterField("bar",
-				func(ctx context.Context, i Item) string {
-					return i.String
-				},
-			),
-		)
+		if expensive {
+			inner.FieldFunc("innerConnectionWithFilter", func() []Item {
+				return items
+			}, schemabuilder.Paginated,
+				schemabuilder.FilterField("foo",
```

Comment:
I like this but instead of inferring from context can we make this explicit?

Reply:
Yup, will's PR #281 should do it automatically

Reply:
So looking back I think #281  will enable the option to allow us to make it explicitly, and I can piggy back off the logic. I think the `FilterField` function in `graphql/schemabuilder/types.go` would just need to have an argument to pass the `expensive` flag in, and it should work